source/endpoints/user.md.erb

```diff
@@ -0,0 +1,78 @@
+# User
+
+## Get user
+
```

Comment:
I would add something saying that user creation and deletion, along with permissions, are done with the subusers endpoint; while this only gets current user data.